cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1611,6 +1613,9 @@ if (testing)
       target_compile_options(${lib} INTERFACE -Wno-deprecated-copy)
     endif()
   endforeach()
+  target_include_directories(gtest INTERFACE ${GTEST_INCLUDE_DIR})
```

Comment:
I think we need to add this:
`
target_include_directories(gtest  INTERFACE ${GTEST_INCLUDE_DIR})
target_include_directories(gtest_main  INTERFACE ${GTEST_INCLUDE_DIR})
target_include_directories(gmock  INTERFACE ${GTEST_INCLUDE_DIR} ${GMOCK_INCLUDE_DIR})
target_include_directories(gmock_main INTERFACE ${GTEST_INCLUDE_DIR} ${GMOCK_INCLUDE_DIR})`

Reply:
I had a look at the previous failures. There's a bug in CMake when you download a project: You have to create the directories.
It could still be that `gtest_main` also needs the INTERFACE, though, should I add it?